test/index.js

```diff
@@ -1,22 +1,38 @@
 var assert = require('assert');
-var Cookies = require('js-cookie');
 var mock = require('mock-cookie');
 
-require('../src/index.js');
-
 document = new mock.Document();
 
 describe('Cookies', function () {
   describe('#removeAll()', function () {
     it('should remove all cookies', function () {
+      var Cookies = require('../src/index.js');
+
       Cookies.set('a', '1');
       Cookies.set('b', '2');
       assert.equal('1', Cookies.get('a'));
       assert.equal('2', Cookies.get('b'));
 
       Cookies.removeAll();
-      assert.equal(undefined, Cookies.get('a'));
-      assert.equal(undefined, Cookies.get('b'));
+
+      var actual = Cookies.get('a');
+      var expected = undefined;
+      assert.strictEqual(actual, expected);
+
+      actual = Cookies.get('a');
+      expected = undefined;
+      assert.strictEqual(actual, expected);
```

Comment:
I am using `strictEqual` here because using `equal` can make you fall in the following trap in case the tested signature changes to `null`:

```javascript
assert.equal(undefined, null); // pass
assert.strictEqual(undefined, null); // fail
```

Strict equality should always be used in tests, because it is expected that no unintentional bugs happen due to a loose equality.